### PR TITLE
DOC-1893: Added v21.2.6 release notes

### PR DESCRIPTION
**Pull request by nickvigilante**
Fixed DOC-1893

### CODE REVIEW

File: releases/v21.2.6.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+[#75847]: https://github.com/cockroachdb/cockroach/pull/75847
+[#75858]: https://github.com/cockroachdb/cockroach/pull/75858
+[#75893]: https://github.com/cockroachdb/cockroach/pull/75893
+[#75979]: https://github.com/cockroachdb/cockroach/pull/75979
+[#76071]: https://github.com/cockroachdb/cockroach/pull/76071
+[#76250]: https://github.com/cockroachdb/cockroach/pull/76250
+[#76255]: https://github.com/cockroachdb/cockroach/pull/76255
+[#76321]: https://github.com/cockroachdb/cockroach/pull/76321
+[#76324]: https://github.com/cockroachdb/cockroach/pull/76324
+[64156f955]: https://github.com/cockroachdb/cockroach/commit/64156f955
```
Comment (jseldess) [inline releases/v21.2.6.md:73]: Are we linking to these commits somewhere? If so, we should change those links to point to the parent PR and then remove these.

Comment (nickvigilante) [reply releases/v21.2.6.md:73]: This style of link is known as a reference link. The syntax of calling it in-line is [#76255][#76255], and the references you see on lines 70 and below are hidden by default. The script has been generating links in this manner prior to my changes.

File: releases/v21.2.6.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+$ docker pull cockroachdb/cockroach:v21.2.6
+~~~
+
+### Enterprise edition changes
+
+- [Kafka sink](../v21.2/changefeed-sinks.html#kafka) supports larger messages, up to 2GB in size. [#76321][#76321]
+
+### SQL language changes
+
+- Added new [built-in functions](../v21.2/functions-and-operators.html#built-in-functions) called `crdb_internal.revalidate_unique_constraint`, `crdb_internal.revalidate_unique_constraints_in_table`, and `crdb_internal.revalidate_unique_constraints_in_all_tables`, which can be used to revalidate existing [unique constraints](../v21.2/unique.html). The different variations support validation of a single constraint, validation of all unique constraints in a table, and validation of all unique constraints in all tables in the current database, respectively. If any constraint fails validation, the functions will return an error with a hint about which data caused the constraint violation. These violations can then be resolved manually by updating or deleting the rows in violation. This will be useful to users who think they may have been affected by [#73024][#73024]. [#75858][#75858]
```
Comment (nickvigilante) [inline releases/v21.2.6.md:42]: @jseldess FYI

File: releases/v21.2.6.md (hunk 1 above)
Comment (jseldess) [reply releases/v21.2.6.md]: Yep, understood. I'm referring to lines 73 and 74. We can remove those.

Comment (nickvigilante) [reply releases/v21.2.6.md]: Removed the loose commit lines. Thanks!

**Review by lnhsingh — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/13081)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @jseldess, @lnhsingh, and @nickvigilante)

___
*[_config_base.yml, line 87 at r2](https://reviewable.io/reviews/cockroachdb/docs/13081#-MwXMJzGDVfmaFDjZPAm:-MwXMJzGDVfmaFDjZPAn:bfxnzjl) ([raw file](https://github.com/cockroachdb/docs/blob/520da4c26d46790bf3fdf835edbbe5fe480ec771/_config_base.yml#L87)):*
> ```YAML
>     version: v21.1.15
>   v21.2:
>     build_time: 2022/02/07 11:00:26 (go1.16.6)
> ```

This date needs to be updated to today
___
*[_config_base.yml, line 90 at r2](https://reviewable.io/reviews/cockroachdb/docs/13081#-MwXMN0A2fPFu_2UOprm:-MwXMN0A2fPFu_2UOprn:b-ya81ng) ([raw file](https://github.com/cockroachdb/docs/blob/520da4c26d46790bf3fdf835edbbe5fe480ec771/_config_base.yml#L90)):*
> ```YAML
>     docker_image: cockroachdb/cockroach
>     name: v21.2.6
>     start_time: 2022-02-07 11:01:26.34274101 +0000 UTC
> ```

Same as above
___
*[releases/v21.2.6.md, line 49 at r2](https://reviewable.io/reviews/cockroachdb/docs/13081#-MwXMpUgDgIZLbCPP8xD:-MwXMpUh9S64n_-OdtVN:b-wunx86) ([raw file](https://github.com/cockroachdb/docs/blob/520da4c26d46790bf3fdf835edbbe5fe480ec771/releases/v21.2.6.md#L49)):*
> ```Markdown
> ### Bug fixes
> 
> - A sequence is created when a column is defined as [`SERIAL`](../v21.2/serial.html) type and the `serial_normalization` session variable is set to `sql_sequence`. In this case, the sequence is owned by the column and the table where the column exists. The sequence should be dropped when the owner table/column is dropped, which is the PostgreSQL behavior. However, CockroachDB never set ownership information correctly but only dependency relationship. Thus, the sequence stays even the owner table/column does not exist anymore. This fix assigns correct ownership information to sequence descriptor and column descriptor to align with PostgreSQL behavior. [#75704][#75704]
> ```

Please rework this bullet - I think this release note needs to be reworked so that the information about the fix is in the beginning. So it should be something like "Fixed a bug where CRDB never set ownership... " and then include the rest of the info, since the rest is important to understanding what the fix is for.
___
*[releases/v21.2.6.md, line 51 at r2](https://reviewable.io/reviews/cockroachdb/docs/13081#-MwXNd_J7F1sESbI9oPw:-MwXNd_KBGWhPRsJ6_69:b-9h7nzz) ([raw file](https://github.com/cockroachdb/docs/blob/520da4c26d46790bf3fdf835edbbe5fe480ec771/releases/v21.2.6.md#L51)):*
> ```Markdown
> - A sequence is created when a column is defined as [`SERIAL`](../v21.2/serial.html) type and the `serial_normalization` session variable is set to `sql_sequence`. In this case, the sequence is owned by the column and the table where the column exists. The sequence should be dropped when the owner table/column is dropped, which is the PostgreSQL behavior. However, CockroachDB never set ownership information correctly but only dependency relationship. Thus, the sequence stays even the owner table/column does not exist anymore. This fix assigns correct ownership information to sequence descriptor and column descriptor to align with PostgreSQL behavior. [#75704][#75704]
> - Fixed a bug that could cause nodes to crash when truncating abnormally large Raft logs. [#75979][#75979]
> - The [DB Console](../v21.2/ui-overview.html) [Databases](../v21.2/ui-databases-page.html) page now shows stable, consistent values for database sizes. [#76324][#76324]
> ```

"Databases" should be bolded since it's the name of a page
___
*[releases/v21.2.6.md, line 55 at r2](https://reviewable.io/reviews/cockroachdb/docs/13081#-MwXO10bAhuXhBh9UjHb:-MwXO10bAhuXhBh9UjHc:bwatcg0) ([raw file](https://github.com/cockroachdb/docs/blob/520da4c26d46790bf3fdf835edbbe5fe480ec771/releases/v21.2.6.md#L55)):*
> ```Markdown
> ### Performance improvements
> 
> - Sorting data with bytes-like types ([`BYTES`](../v21.2/bytes.html), [`STRING`](../v21.2/string.html), [`JSON`/`JSONB`](../v21.2/jsonb.html), [`UUID`](../v21.2/uuid.html)) when [`LIMIT`](../v21.2/limit-offset.html) clause is specified has now become more predictable. [#75847][#75847]
> ```

nit:  "..when the LIMIT"

**Review by nickvigilante — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/13081)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @jseldess and @lnhsingh)

___
*[_config_base.yml, line 87 at r2](https://reviewable.io/reviews/cockroachdb/docs/13081#-MwXMJzGDVfmaFDjZPAm:-MwXQGso6_uvbd_2W3gq:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/520da4c26d46790bf3fdf835edbbe5fe480ec771/_config_base.yml#L87)):*
<details><summary><i>Previously, lnhsingh (Lauren Hirata Singh) wrote…</i></summary><blockquote>

This date needs to be updated to today
</blockquote></details>

Done.
___
*[_config_base.yml, line 90 at r2](https://reviewable.io/reviews/cockroachdb/docs/13081#-MwXMN0A2fPFu_2UOprm:-MwXQHlq-_gAXhW3Enl2:bd1fu5x) ([raw file](https://github.com/cockroachdb/docs/blob/520da4c26d46790bf3fdf835edbbe5fe480ec771/_config_base.yml#L90)):*
<details><summary><i>Previously, lnhsingh (Lauren Hirata Singh) wrote…</i></summary><blockquote>

Same as above
</blockquote></details>

Done. This date is arbitrary so I'm going to find a way to replace/remove it.
___
*[releases/v21.2.6.md, line 49 at r2](https://reviewable.io/reviews/cockroachdb/docs/13081#-MwXMpUgDgIZLbCPP8xD:-MwXQP-H0SZ2zv2Bquo0:b-r75l0) ([raw file](https://github.com/cockroachdb/docs/blob/520da4c26d46790bf3fdf835edbbe5fe480ec771/releases/v21.2.6.md#L49)):*
<details><summary><i>Previously, lnhsingh (Lauren Hirata Singh) wrote…</i></summary><blockquote>

Please rework this bullet - I think this release note needs to be reworked so that the information about the fix is in the beginning. So it should be something like "Fixed a bug where CRDB never set ownership... " and then include the rest of the info, since the rest is important to understanding what the fix is for.
</blockquote></details>

I pushed it, but do you maybe have 10 minutes to go through it with me at some point today if it still looks funky?
___
*[releases/v21.2.6.md, line 51 at r2](https://reviewable.io/reviews/cockroachdb/docs/13081#-MwXNd_J7F1sESbI9oPw:-MwXQWKdBYtrQV4NdGD4:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/520da4c26d46790bf3fdf835edbbe5fe480ec771/releases/v21.2.6.md#L51)):*
<details><summary><i>Previously, lnhsingh (Lauren Hirata Singh) wrote…</i></summary><blockquote>

"Databases" should be bolded since it's the name of a page
</blockquote></details>

Done.

**Review by lnhsingh — COMMENTED** [review]
LGTM

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/13081#-:-MwY2Hb84jFNhnMYqYqp:bb6ywmx)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @jseldess and @lnhsingh)

**Pull request merged**